frontend/src/shared.scss

```diff
@@ -31,6 +31,14 @@
   box-shadow: 0px 0px 0px 3px rgba(255, 48, 97, 0.25);
 }
 
+.visdomLogo {
+  cursor: pointer;
+  &:hover,
+  &.selected {
+    filter: brightness(0.8);
+  }
```

Comment:
I'm not quite sure if the filter should stay when selected
To my eye, the dim logo doesn't look quite as good

Reply:
In my opinion the dimmed logo is good while it's hovered but I agree that it looks a bit off when it stays dimmed when selected. But I think there should be some other indication that you're on this page, maybe something similar to the other buttons on the navbar?

Also, does the user know that the logo in the corner is clickable button to the projects-page or should there be another way to get there/some indication (other than the hover) that you can indeed click it?

Reply:
>some other indication that you're on this page

The lack of indication on the other sidebar items could be taken as indication

(It might even make sense to not show the sidebar on the overview page, as all those items are roadmap specific.
Then the overview page would be the home page for the whole application, and dashboard page would be the home page for a roadmap.)

>does the user know that the logo in the corner is clickable button to the projects-page

I think the logo at the top left is quite idiomatic as link to the "home" page. This might be more obvious if the user would be directed on that page on login.

Maybe the "home" page resolution after login could be taking the first matching:
1. redirect in url
2. default roadmap
3. the overview page

Reply:
> It might even make sense to not show the sidebar on the overview page, as all those items are roadmap specific.

That would make sense, it would separate homepage from roadmap-specific pages nicely. I think this is worth trying, it could look and feel really good. I was playing around a bit with this idea, thoughts?


![idea](https://user-images.githubusercontent.com/80753884/127267464-a40df9e5-94c7-4159-8f31-7bf89aeef0cd.png)

You'd probably have to change the functionality a bit though in this case since you really can't have a disabled open -button on your selected roadmap if the only way to get to roadmaps is the open -button. Maybe going to the homepage would set your selected roadmap to none.

> 1. redirect in url 2.default roadmap 3. the overview page

That sounds like a good idea

Reply:
For now I removed the selected filter effect on the Visdom logo.
The overview page works better without the sidebar 👍 I however think the top bar should stay the same, perhaps without the project selector dropdown.